training-net/src/Controllers/MoviesController.cs

```
@@ -123,7 +129,7 @@ public IActionResult Delete(int id)
         [HttpPost("DeleteConfirmation")]
         public IActionResult DeleteConfirmation(MovieViewModel mvm)
         {
-            UnitOfWork.MovieRepository.Remove(UnitOfWork.MovieRepository.Get(mvm.ID));
+            UnitOfWork.MovieRepository.Remove(UnitOfWork.MovieRepository.Get(mvm.ID ?? default(int)));
```

why this `default(int)`? If mvm is null, we can throw an exception, but i dont understand why send a default int to remove